### PR TITLE
LPS-196231 MariaDB Driver | master

### PR DESCRIPTION
**Pull request by Kvale1733**
https://liferay.atlassian.net/browse/LPS-196231

Tested this here on all versions of MariaDB and MySQL and found no regressions.
https://github.com/Kvale1733/liferay-portal/pull/415. We can just check relevant for this pull.

Just needed to add a property to the MySQL url to work with the MariaDB driver and suppress the warning thrown here
https://liferay.atlassian.net/browse/LPS-196480 as discussed here:
https://liferay.slack.com/archives/GLCFMMZQW/p1695064710439229

### CODE REVIEW

**Comment by Sharryxue** [comment]
ci:test:stable

**Comment by Sharryxue** [comment]
QA Passed.
No related issue found in [ci:test:relevant](https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2427776113).

Could you please help with adding the label @slnn, thanks in advance.

File: test.properties [hunk 1]
```diff
@@ -439,7 +439,7 @@
     database.mysql.service.cmd.stop=mysqld stop
     database.mysql.service.executable=service
     database.mysql.username=root
-    database.mysql.url=jdbc:mysql://${database.mysql.host}/${database.mysql.schema}?characterEncoding=UTF-8&dontTrackOpenResources=true&holdResultsOpenOverStatementClose=true&serverTimezone=GMT&useFastDateParsing=false&useUnicode=true
+    database.mysql.url=jdbc:mysql://${database.mysql.host}/${database.mysql.schema}?characterEncoding=UTF-8&dontTrackOpenResources=true&holdResultsOpenOverStatementClose=true&serverTimezone=GMT&useFastDateParsing=false&useUnicode=true?permitMysqlScheme
```
Comment (dantewang) [inline test.properties:442]: @Kvale1733 Can you confirm whether it should be `?permitMysqlScheme` or `&permitMysqlScheme` or both work? Usually `?` is the indicator of the beginning of the query string path in a URL, while the parameters are separated by `&`, and this URL already has a lot of parameters.

cc @vicnate5

Comment (dantewang) [reply test.properties:442]: @Kvale1733 Also, could you make it more specific in the commit message about the reason that we need to add this parameter? I mean, just looking at these properties, it seems to be used together with MySQL's own driver. So we'd better point out if there are tests that use this URL, but with the MariaDB driver name in `database.mysql.driver` or something like that.

Comment (Kvale1733) [reply test.properties:442]: On it.

**Comment by Kvale1733** [comment]
@dantewang Both work, but it does make more sens to use '&', so I switch it. Also changed the commit message to better reflect what that parameter is for as stated in https://mariadb.com/kb/en/about-mariadb-connector-j/#jdbcmysql-scheme-compatibility

**Comment by Kvale1733** [comment]
ci:test:sf

**Comment by Kvale1733** [comment]
ci:test:relevant

**Review by dantewang — APPROVED** [review]

**Pull request closed without merging**